### DISCUSSION TITLE
New Demo: Isometric Diagram

### DISCUSSION THREAD

**Discussion by kumilingus** [clientIO/joint#2205] [News]
Did you know that you can **display a 2D diagram in an isometric projection** while retaining most of the interactivity? This can be accomplished by using an isometric transformation applied to the SVG of your document and a specific way of structuring the shapes. This is an [example](https://www.jointjs.com/demos/isometric-diagram) that proves it.

_An article with an example and step-by-step instructions will follow soon!_

The demo also shows how to **restrict element movement** and resizing so that **no 2 elements overlap**.

<img width="922" alt="image" src="https://github.com/clientIO/joint/assets/3967880/2081ddd1-deb0-4815-b9b7-ad436886bf31">

**Comment by alexandernst** [comment]
WoW!!
